docs/installation.md

```diff
@@ -1,101 +1,50 @@
 # Installation
 
 Before starting ensure [requirements](requirements.md) are met
-## GitHub Actions (recommended)
 
-<img src="assets/images/pipeline.png">
+## Initial installation
 
-* deploy Domain Protect in your AWS Organization
-* no need to clone or fork Domain Protect
-* internal / private deployment repository to protect sensitive information
-* uses OpenID Connect - no IAM user with long-lived access keys
-* update to latest version of Domain Protect any time by running pipeline
+* Include the following code snippet to your code
 
-Follow detailed instructions in separate [Domain Protect Deploy](https://github.com/domain-protect/terraform-aws-domain-protect-deploy) repository with GitHub Actions deployment workflow
-
-## Atlantis
-
-Per [official documentation](https://www.runatlantis.io/), Atlantis is an application for automating Terraform via pull requests
-
-* For atlantis to work, you can simply create a new stack (folder) in your environment with the name `domain-protect`. You will then make the necessary changes to the `atlantis.yaml` file, so atlantis knows where all the terraform code is from.
-
-Example changes to `atlantis.yaml`:
-
-```yaml
-- name: domain-protect-security-audit
-  dir: security/domain-protect
-  workspace: security-audit
-  workflow: security-audit
-  terraform_version: 1.3.1
-```
-
-* In the `domain-protect` folder, you can copy the content of `main.tf` file from this repo. For each of the module, make sure you change the source to this repo so you can reuse the module here
-
-For example, update:
-
-```terraform
-module "kms" {
-  source  = "./modules/kms"
-  project = var.project
-  region  = var.region
-}
 ```
-
-to
-
-```terraform
-module "kms" {
-  source  =  "git::https://github.com/domain-protect/terraform-aws-domain-protect.git//modules/kms"
-  project = var.project
-  region  = var.region
+module "domain_protect" {
+  source  = "domain-protect/domain-protect/aws"
+  version = "1.0.0"
+
+  environment              = var.environment
```

Comment:
Instead of redefining each variable here, for the sake of the example, why not simply use the hard values here?

E.g.

```suggestion
  environment              = "prod"
```

Reply:
updated